### PR TITLE
Update agency v2

### PR DESCRIPTION
**Pull request by russellbodine**
Must add FY TOTAL BUDGET AUTHORITY to DB

### CODE REVIEW

File: usaspending_api/references/views_v2.py [hunk 1]
```diff
@@ -24,17 +25,53 @@ def get(self, request, pk, format=None):
...
+            obligated_amount=Sum('obligations_incurred_total_by_tas_cpe'),
+            outlay_amount=Sum('gross_outlay_amount_by_tas_cpe'))
+
         submission = queryset.first()
         if submission is None:
             return Response(response)
-        active_fiscal_year = submission.fiscal_year
+
+        # get the overall total government budget authority (to craft a budget authority percentage)
+        queryset2 = OverallTotals.objects.all()
```
Comment (nmonga91) [inline usaspending_api/references/views_v2.py]: We should probably name this something other than `queryset2` to be a bit more descriptive. Something like `total_budget_authority_queryset` or something along those lines.

File: usaspending_api/references/views_v2.py [hunk 2]
```diff
@@ -24,17 +25,53 @@ def get(self, request, pk, format=None):
...
+        queryset2 = queryset2.filter(fiscal_year=active_fiscal_year)
+
+        submission2 = queryset2.first()
 
         # craft response
-        response['results']['agency_name'] = toptier_agency.name
+        response['results']['total_budget_authority_amount'] = str(submission2.total_budget_authority)
```
Comment (nmonga91) [inline usaspending_api/references/views_v2.py]: Here we could make it look a bit better when compiling the response by doing something like:
```
response['results'] = { 'total_budget_authority_amount': str(submission2.total_budget_authority),
                                      'budget_authority_amount': str(submission.budget_authority_amount),
                                      ...
                                     }
```

With better spacing than my example 😄

**Comment by russellbodine** [comment]
- [ ] OverallTotal Table

File: usaspending_api/references/views_v2.py (hunk 1 above)
Comment (russellbodine) [inline usaspending_api/references/views_v2.py]: done!

**Pull request merged**